development/dev.env

```diff
@@ -1,4 +1,4 @@
-NAUTOBOT_ALLOWED_HOSTS=".localhost 127.0.0.1 [::1]"
+NAUTOBOT_ALLOWED_HOSTS=".localhost 127.0.0.1 [::1] *"
```

Comment:
Not a fan of adding `*` to NAUTOBOT_ALLOWED_HOSTS - what was the reason here?

Reply:
Accidental commit from when I was trying to get the OpenTelemetry autotooling working